README.md

```diff
@@ -24,26 +24,7 @@ The [contributing guide](https://github.com/cbrnr/sleepecg/blob/main/CONTRIBUTIN
 
 
 ## Dataset reading
```

Comment:
Should we rename this to just "Datasets"? This would look nicer in the toolbar and I think it is still pretty accurate.